ethereum/core/build.gradle

```diff
@@ -84,7 +88,7 @@ dependencies {
 }
 
 configurations { testArtifacts }
-task testJar (type: Jar) {
+task testJar(type: Jar) {
```

Comment:
Pleas limit gradle changes to just the minimum needed.  Gradle formatting can be done in another PR.

Reply:
indeed. Sorry, I've been trying to catch myself on those. Let me know if you see others. I have been trying to trim out unneeded changes, but looks like I've done a poor job of it. Apologies.